dbt/models/marts/weekly/beneficiaires.sql

```diff
@@ -0,0 +1,50 @@
+select
+    etp.identifiant_salarie,
+    etp.id_annexe_financiere,
+    etp.emi_sme_annee,
+    etp.majoration_brsa,
+    etp.salarie_brsa,
+    etp.type_structure,
+    etp.type_structure_emplois,
+    etp.structure_id_siae,
+    etp.structure_denomination,
+    etp.commune_structure,
+    etp.code_insee_structure,
+    etp.siret_structure,
+    etp.epci_structure,
+    etp.nom_departement_af,
+    etp.nom_region_af,
+    etp.hash_nir,
+    etp.genre_salarie,
+    salarie.salarie_adr_zrr_classement as classement_zrr,
+    case
+        when salarie.salarie_adr_qpv_type = 'NQP' then 'Adresse non QPV'
+        when salarie.salarie_adr_qpv_type = 'QP' then 'Adresse en QPV'
+        else 'Statut QPV non renseigné'
+    end                                as adresse_qpv,
+    sum(etp.nombre_heures_travaillees) as nombre_heures_travaillees
+from
+    {{ ref('etp_par_salarie') }} as etp
+left join {{ ref('fluxIAE_Salarie_v2') }} as salarie
```

Comment:
il y a déjà une jointure sur cette table dans `etp_par_salarie`, why not ajouter zrr et qpv directement ds `etp_par_salarie` et s'éviter cette jointure ?

Reply:
ouais pas sûre de ce qui est le mieux en terme de perf mais ok !